ros/src/util/packages/autoware_launcher/documents/demos/rosbag.md

```diff
@@ -0,0 +1,36 @@
+# Rosbag Demo
+
+## Demo Data
+1. Download the following files from https://www.autoware.ai/ ([Mirror](https://drive.google.com/open?id=1ZwNQIJolJ2ogzpNprh89jCj6NocLi78f)).<br>
+  * https://www.autoware.ai/sample/sample_moriyama_data.tar.gz
+  * https://www.autoware.ai/sample/sample_moriyama_150324.tar.gz
+
+1. Make `~/.autoware` directory and extract the demo data.<br>
```

Comment:
1. Create the `.autoware` directory in your home directory.
1. Inside this, extract the previously downloaded files.